esp/esp/program/modules/handlers/programprintables.py

```diff
@@ -934,26 +1074,35 @@ def getSchedule(program, user, schedule_type=None, room_numbers=True, include_da
             elif user.isVolunteer():
                 schedule_type = u'Volunteer'
 
-
+        include_roles = None
```

Comment:
Why `None` instead of `False`?

Reply:
bc the combination of booleans and django templates are confusing